src/Documentation/Advanced-Concepts/Powershell-Client.md

```diff
@@ -137,7 +133,11 @@ PS> $grainId = [System.Guid]::Parse("A4CF7B5D-9606-446D-ACE9-C900AC6BA3AD")
 
 PS> $grain = Get-Grain -GrainType $grainInterfaceType -GuidKey $grainId
 
-PS> $message = $grain.SayHelloTo("Gutemberg").Result
+PS> $sayHelloTask = $grain.SayHelloTo("Gutemberg")
+
+PS> $sayHelloTask.Wait();
+
+PS> $message = $sayHelloTask.Result
```

Comment:
I will try to double check.  BTW it looks like psgallery does not have the module published